electron/main.ts

```diff
@@ -134,6 +135,11 @@ async function createWindow(): Promise<BrowserWindow> {
     }
   }
 
+  if (isSteamDeckInGamingMode()) {
```

Comment:
This should be above line 111 `if(configStore.has('window-props') ...` and if should be replaced on line 111 with else if.

Also skip saving the size in GameMode. Saving is in `mainWindow.on('close', async (e) => { ...`